core/src/main/java/org/elasticsearch/index/IndexService.java

```diff
@@ -538,8 +537,9 @@ public void handle(ShardLock lock) {
         }
     }
 
-    public Settings getIndexSettings() {
-        return indexSettings;
+    @Override
+    public Settings indexSettings() {
```

Comment:
oh I see ok - I think we should remove that at least in master since I think it's still there but shouldn't